openapi/docs/description.md

```diff
@@ -364,6 +364,160 @@ Use these card numbers to test successful transactions as well as various error
   </tr>
 </table>
 
+#### Testing Disputes
```

Comment:
Hm, it looks like this section is repeating the dispute testing section above. What is the reason for that? 
If we need it can you please change it the same way I suggested in the one above?